pkgs/development/libraries/abseil-cpp/default.nix

```diff
@@ -9,24 +9,15 @@
 
 stdenv.mkDerivation rec {
   pname = "abseil-cpp";
-  version = "20210324.2";
+  version = "20211102.0";
 
   src = fetchFromGitHub {
     owner = "abseil";
     repo = "abseil-cpp";
     rev = version;
-    sha256 = "0g9rbhk3mwjdfxk7cscd04vm8fphd5flz9yykpgvyy1nwa34zk3x";
+    sha256 = "sha256-sSXT6D4JSrk3dA7kVaxfKkzOMBpqXQb0WbMYWG+nGwk=";
   };
 
-  patches = [
-    # Use CMAKE_INSTALL_FULL_{LIBDIR,INCLUDEDIR}
-    # https://github.com/abseil/abseil-cpp/pull/963
-    (fetchpatch {
```

Comment:
`fetchpatch` should be removed form attributes.

Reply:
I actually need to restore the old version.

Dont remember which package, but other packages aren't api compatible with it

Reply:
found it,
```
error: builder for '/nix/store/yjssm66bj004hhviri4akwc7gxwj3qcq-tensorflow-lite-2.5.0.drv' failed with exit code 2;
...
       > cordz_info.cc:(.text.unlikely+0x31): undefined reference to `absl::lts_20211102::Mutex::~Mutex()'
       > collect2: error: ld returned 1 exit status